login-workflow/src/auth-shared/index.ts

```diff
@@ -23,7 +23,6 @@ export { AuthUIContextProvider } from './contexts/AuthUIContextProvider';
 export type {
     AuthUIContextProviderProps,
     RegistrationUIActions,
-    AuthUIActions,
```

Comment:
If you're going this route instead of setting up a new index (the new index is the better route), then make this a comment instead of removing the line entirely so we know to come back and change it later if needed (maybe add a TODO comment as well)